src/contexts/RecordingSelectionContext.ts

```diff
@@ -102,8 +102,8 @@ export const useTimeRange = () => {
         })
     }, [recordingSelectionDispatch])
     return {
-        visibleTimeStartSeconds: recordingSelection.visibleTimeStartSeconds,
-        visibleTimeEndSeconds: recordingSelection.visibleTimeEndSeconds,
+        visibleTimeStartSeconds: recordingSelection.visibleTimeStartSeconds !== undefined ? recordingSelection.visibleTimeStartSeconds - timestampOffset : undefined,
+        visibleTimeEndSeconds: recordingSelection.visibleTimeEndSeconds !== undefined ? recordingSelection.visibleTimeEndSeconds - timestampOffset : undefined,
```

Comment:
I don't follow why the timestamp offset is being subtracted from these values.

Reply:
This is where we should chat in person.